utils/binpath.go

```diff
@@ -28,6 +29,10 @@ var (
 	ExternalBinaryPaths = []string{"/usr/sbin", "/sbin", "/usr/local/sbin"}
 )
 
+var (
+	BrewSpecificPaths = [string]{"/usr/local/opt"}
```

Comment:
Not sure if this `[string]` syntax is also acceptable, but `[]string` seems to be the right or at least preferred one.